MobileCenter/MobileCenter/Internals/Channel/MSLogManagerDelegate.h

```diff
@@ -9,8 +9,30 @@
  *  On processing log callback.
  *
  *  @param log      log.
+ *  @param internalId An internal Id that can be used to keep track of logs.
  *  @param priority priority.
  */
-- (void)onProcessingLog:(id<MSLog>)log withPriority:(MSPriority)priority;
+- (void)onProcessingLog:(id<MSLog>)log withInternalId:(NSString *)internalId andPriority:(MSPriority)priority;
```

Comment:
One suggestion for this delegate.
Can we have different names something like onPersistingLog, onFinishedPersistingLog?
It can help out to get out of confusions that I had. 😄